frontend/src/stores/user.js

```diff
@@ -23,8 +25,22 @@ class UserStore {
         '3': '',
     };
     pinCodeIsSet = false;
-    balance = 0;
+    balance = 100;
 
+    get isAuthenticated() {
+        // user is authenticated if there exists a token, otherwise they are guests
+        return !!this.token;
```

Comment:
We didn't plan to store the token to be accessible for frontend.

Reply:
yes, these are "for now" place holder implementation. Its quite annoying also for me to have to constantly login